README.md

```diff
@@ -50,4 +50,4 @@ with a workflow :
 
 - Going Open Source brings and requires a change of mentality within development teams.
 
-![Worflow step 4](./images/workflow-step4.png)
\ No newline at end of file
+![Worflow step 4](./images/workflow-step4.png)
```

Comment:
Worklfow?

Thank you gor your pr @JLou  😀

Reply:
I didn't touch that line, it's the absence of new line at the end making git confused. I can fix it though

Reply:
I fixed all the instances of _worflow_. Should be good to merge now!